### ISSUE TITLE
the MemAvailable information incorrect ( linux kernel < 3.14)

### ISSUE THREAD

**Issue by mikezsin** [shirou/gopsutil#633]
**Describe the bug**
if !memavail {
  ret.Available = ret.Free + ret.Buffers + ret.Cached
  }
it is wrong way  to calculate MemAvailable 

**To Reproduce**

for example, postgres shared_buffers may be reported under "Cached" 

**Expected behavior**
bash version
```bash
LOW_WATERMARK=$(awk '$1 == "low" {LOW_WATERMARK += $2} END {print LOW_WATERMARK * 4096}' /proc/zoneinfo)

MEMINFO=$(</proc/meminfo)

MEMINFO_MEMFREE=$(echo "${MEMINFO}" | awk '$1 == "MemFree:" {print $2 * 1024}')
MEMINFO_FILE=$(echo "${MEMINFO}" | awk '{MEMINFO[$1]=$2} END {print (MEMINFO["Active(file):"] + MEMINFO["Inactive(file):"]) * 1024}')
MEMINFO_SRECLAIMABLE=$(echo "${MEMINFO}" | awk '$1 == "SReclaimable:" {print $2 * 1024}')

MEMINFO_MEMAVAILABLE=$((
  MEMINFO_MEMFREE - LOW_WATERMARK
  + MEMINFO_FILE - ((MEMINFO_FILE/2) < LOW_WATERMARK ? (MEMINFO_FILE/2) : LOW_WATERMARK)
  + MEMINFO_SRECLAIMABLE - ((MEMINFO_SRECLAIMABLE/2) < LOW_WATERMARK ? (MEMINFO_SRECLAIMABLE/2) : LOW_WATERMARK)
))

if [[ "${MEMINFO_MEMAVAILABLE}" -le 0 ]]
then
  MEMINFO_MEMAVAILABLE=0
fi
```

**Environment (please complete the following information):**
 - [x] Linux: [< 3.14]

**Comment by Lomanic** [comment]
This will be fixed when #626 is merged.

**Comment by Lomanic** [comment]
PR was merged.
